nss-day-cohort-48/the-worst-site-ever-fandango
# added navbar guide

added a beautiful guide for the navbar

steps to test:
1. fetch/code ./serve
2. check that there is now a directory called "/img" in the main directory (in vs code)
3. check that the img is in the index.html right under the navbar (in vs code)
4. check that the navbar guide displays properly in chrome right under the navbar
